source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,61 @@
+---
+title: EU exit business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+owner_slack: "#govuk-euexit-ready"
+last_reviewed_on: 2019-06-19
+review_in: 6 months
+---
+
+The [EU exit business readiness finder](https://www.gov.uk/find-eu-exit-guidance-business) is a special finder built to help businesses prepare for EU exit. A number of proprietary modifications have been made to the normal [finder-frontend](/apps/finder-frontend.html) model, which we've documented here.
+
+For a list of all of the YAML files that drive the finder and its related pages, see: [Publish changes to the business readiness finder](/manual/business-readiness-publish-changes.html).
+
+## Overview
+
+The full journey for the business readiness finder is made up of the following content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+4. The email signup page: [`/find-eu-exit-guidance-business/email-signup`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup)
+
+Editors can request changes to the finder via a [form](https://support.integration.publishing.service.gov.uk/eu_exit_business_readiness_request/new) in the [Support](https://github.com/alphagov/support) app.
+
+### Start page
+
+A normal page which can be updated in Mainstream Publisher.
+
+### Q&A
+
+The Q&A is [documented in finder-frontend](https://github.com/alphagov/finder-frontend/blob/master/docs/q-and-a.md).
+
+### Finder
```

Comment:
I think this whole section should be in  the finder-frontend README, as it's very specific to how finder frontend works.

The documentation in the developer should really be limited to a general overview and support tasks perhaps with links to the detailed documentation in the repo.